tests/models/test_token_classification_with_seq2seq_encoder_and_crf.py

```diff
@@ -0,0 +1,258 @@
+import json
+
+import pytest
+import torch
+import transformers
+from pytorch_ie.taskmodules import TransformerTokenClassificationTaskModule
+from transformers.modeling_outputs import BaseModelOutputWithPoolingAndCrossAttentions
+
+from pie_models.models import TokenClassificationModelWithSeq2SeqEncoderAndCrf
+from tests import FIXTURES_ROOT
+
+DUMP_FIXTURES = False
+FIXTURES_TASKMODULE_DATA_PATH = (
+    FIXTURES_ROOT / "taskmodules" / "TransformerTokenClassificationTaskModule"
```

Comment:
can you change this to 
```suggestion
    FIXTURES_ROOT / "taskmodules" / "token_classification_with_seq2seq_encoder_and_crf"
```